CBLClient/Apps/CBLTestServer-Dotnet/TestServer/DatabaseMethods.cs

```diff
@@ -528,20 +528,30 @@ public static async Task AsyncWith<T>([NotNull]IReadOnlyDictionary<string, objec
             });
             response.WriteEmptyBody();
         }
+
         internal static void DatabaseCopy([NotNull] NameValueCollection args,
                                                   [NotNull] IReadOnlyDictionary<string, object> postBody,
                                                   [NotNull] HttpListenerResponse response)
         {
             string dbName = postBody["dbName"].ToString();
-            string dbPath = TestServer.FilePathResolver(postBody["dbPath"].ToString(), true);
-            dbPath = dbPath + "/";
+            string dbPath = postBody["dbPath"].ToString();
             DatabaseConfiguration dbConfig = MemoryMap.Get<DatabaseConfiguration>(postBody["dbConfig"].ToString());
             Database.Copy(dbPath, dbName, dbConfig);
             response.WriteEmptyBody();
         }
 
+        internal static void DatabaseGetPreBuiltDb([NotNull] NameValueCollection args,
+                                                  [NotNull] IReadOnlyDictionary<string, object> postBody,
+                                                  [NotNull] HttpListenerResponse response)
+        {
+            string dbPath = TestServer.FilePathResolver(postBody["dbPath"].ToString());
```

Comment:
Add true at the end, same as whatever is deleted in DatabaseCopy. That is required for database copy

Reply:
didn't get your point